ServerlessLibraryUI/src/components/AppItem.vue

```diff
@@ -69,14 +69,32 @@ export default {
     }
   },
   methods:{
-     showConsentModal(data) {
-        this.isModalVisible = true;
-        this.modalData = data;
-      },
+    showConsentModal(item) {
+      this.isModalVisible = true;
+      this.modalData = item;
+    },
+
+    outboundRepoClick(item) {
+      this.trackEvent('/sample/source', item);
+    },
 
+    trackEvent(eventName, item) {
```

Comment:
Maybe refactor at a later stage to have the logging /trackEvent in a central component